libs/adapters/src/utils/startHealthCheckLoop.ts

```diff
@@ -31,19 +28,19 @@ export function startHealthCheckLoop({
   if (!enabled) {
     return;
   }
+  const log = logger().getLogger(__filename);
   log.info(`starting health check loop for ${service}`);
-  const stat = `service.health.${service}`;
-  // perform a loop that invokes 'checkFn' and sends status to StatsD
+  const key = `service.health.${service}`;
+  // perform a loop that invokes 'checkFn' and sends status to stats
   const loop = async () => {
     const nextCheckAt = Date.now() + PING_INTERVAL;
-    let status = 0;
     try {
       await checkFn();
-      status = 1;
+      stats().on(key);
```

Comment:
Does this not require passing an adapter?

Reply:
No. The adapter is injected in the first call to the port, usually the server bootstrapping file (index).